nobl9/resource_agent_test.go

```diff
@@ -173,31 +173,6 @@ resource "nobl9_agent" "%s" {
 `, name, name, testProject)
 }
 
-func testCloudWatchDirectBeta(name string) string {
```

Comment:
actually, why this is being removed?

Reply:
t wasn't used in the agent tests, it was only there for SLO tests, which are now rewritten